### ISSUE TITLE
How to handle validation when a component is removed?

### ISSUE THREAD

**Issue by thuhaile** [stsrki/Blazorise#1345]
Hi,

My codes have a set of components created in a for each loop and being validated. My problem is I have a delete button that can delete any set of components and I click delete, the set of components are removed but the validations still reference to these deleted components and hence return validation error. Any suggestion to workaround this issue?

```
<Validations @ref="validations" Mode="ValidationMode.Manual">
   foreach (var component in components)
   {
      <Validation>
         <ComponentA></ComponentA>
      </Validation>
      <Button @onclick="@(() => components.Remove(compoent))">Remove Component</Button>
   }
</Validations>
```

**Comment by stsrki** [comment]
I guess you're using v0.9.1.x? This was fixed for v0.9.2 and s currently only available in preview: https://github.com/stsrki/Blazorise#try-preview

**Comment by thuhaile** [comment]
Thank you. I’ll try the preview 0.9.2.
